spec/requests/sitemaps_spec.rb

```diff
@@ -40,5 +40,39 @@
       expect(response.body).not_to include(articles.last.path)
       expect(response.media_type).to eq("application/xml")
     end
+
+    it "renders most recent posts if /sitemap-posts" do
+      create_list(:article, 8)
+      get "/sitemap-posts.xml"
+      expect(response.body).to include(Article.order("published_at DESC").first.path)
+      expect(response.body).not_to include(Article.order("published_at DESC").last.path)
+    end
+
+    it "renders second page if /sitemap-posts-1" do
```

Comment:
_**Quick, non-blocking suggestion:**_
```suggestion
    it "renders second page if /sitemap-posts-1", :aggregate_failures do
```